scripts/docker/yscope-velox-builder.dockerfile

```
@@ -0,0 +1,81 @@
+COPY scripts /velox/scripts/
+
+# Set timezone to avoid interactive prompts during apt installations
+ENV TZ=Etc/UTC
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install CMake 3.28.3 using official binary installer
+# NOTE: setup-ubuntu.sh installs cmake via pip, but pip-installed cmake is not reliably found
+# in container environments, causing FastFloat library build failures. Using the official
+# binary installer ensures cmake is available and dependencies build correctly in the current
+# CI containerized environment.
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.28.3/cmake-3.28.3-linux-x86_64.sh \
```

Hadolint (2.14.0)

[info] Avoid use of wget without progress bar. Use `wget --progress=dot:giga <url>`. Or consider using `-q` or `-nv` (shorthands for `--quiet` or `--no-verbose`).

(DL3047)